the Lens.js

```diff
@@ -0,0 +1,215 @@
+{
+	"translatorID": "b5f3b896-df57-4c68-8841-ea2171ee7419",
+	"label": "the Lens",
+	"creator": "Laurent Lhuillier",
+	"target": "^https?://www\\.lens\\.org",
+	"minVersion": "4.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2018-06-21 09:15:02"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+	
+	the Lens translator
+	
+	This file is part of Zotero.
+	
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+	
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU Affero General Public License for more details.
+	
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero.  If not, see <http://www.gnu.org/licenses/>.
+	
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if(url.indexOf("search?") !== -1) {
+		return "multiple";
+	}
+		
+	if (url.indexOf("patent") !== -1 && getTitle(doc)) {
+		return "patent";
```

Comment:
What about some of the other subpages like https://www.lens.org/lens/patent/109-941-370-099-325/citations  ?